.github/workflows/main.yml

```diff
@@ -16,4 +16,4 @@ jobs:
       - run: pip install poetry==1.1.11
```

Comment:
Really old version of poetry, probably why the pipeline PR pipeline is failing, let's bump this.